cxAstScan/services/CleanUpRunner.ts

```diff
@@ -17,13 +17,17 @@ export class CleanUpRunner {
         const cxScanConfig = getConfiguration();
         const wrapper = new CxWrapper(cxScanConfig);
 
-        const data = await fs.readFile(getLogFilename(), 'utf8');
+        const data = await fs.readFile(getLogFilename(), 'utf8').catch((() => {
```

Comment:
Could we check for the existence of the file before reading instead of relying on exception?